tests/test_features/test_rankd.py

```diff
@@ -430,98 +368,99 @@ def test_rank2d_covariance(self):
+                    1.11888744e-02,
+                    -9.24206867e00,
+                    -9.40391134e-01,
+                    -4.15083877e00,
+                    1.53324641e-01,
+                    0.00000000e00,
+                    1.57414282e-18,
+                    -1.85278419e-17,
+                ],
+                [
+                    -9.24206867e00,
+                    7.75916384e03,
+                    7.51290743e02,
+                    3.50393655e03,
+                    -1.32370274e02,
+                    0.00000000e00,
+                    -2.65874531e-15,
+                    -4.86170571e-14,
+                ],
+                [
+                    -9.40391134e-01,
+                    7.51290743e02,
+                    1.90326988e03,
+                    -5.75989570e02,
+                    2.14654498e01,
+                    0.00000000e00,
+                    4.57406096e-17,
+                    0.00000000e00,
+                ],
+                [
+                    -4.15083877e00,
+                    3.50393655e03,
+                    -5.75989570e02,
+                    2.03996306e03,
+                    -7.69178618e01,
+                    0.00000000e00,
+                    -1.97089918e-15,
+                    1.54151644e-14,
+                ],
+                [
+                    1.53324641e-01,
+                    -1.32370274e02,
+                    2.14654498e01,
+                    -7.69178618e01,
+                    3.06649283e00,
+                    0.00000000e00,
+                    -2.89497529e-19,
+                    0.00000000e00,
+                ],
+                [
+                    0.00000000e00,
+                    0.00000000e00,
+                    0.00000000e00,
+                    0.00000000e00,
+                    0.00000000e00,
+                    1.25162973e00,
+                    -3.61871912e-20,
+                    0.00000000e00,
+                ],
+                [
+                    1.57414282e-18,
+                    -2.65874531e-15,
+                    4.57406096e-17,
+                    -1.97089918e-15,
+                    -2.89497529e-19,
+                    -3.61871912e-20,
+                    1.77477184e-02,
+                    4.40026076e-02,
+                ],
+                [
+                    -1.85278419e-17,
+                    -4.86170571e-14,
+                    0.00000000e00,
+                    1.54151644e-14,
+                    0.00000000e00,
+                    0.00000000e00,
+                    4.40026076e-02,
+                    2.40547588e00,
                 ],
             ]
         )
 
         assert hasattr(oz, "ranks_")
-        assert oz.ranks_.shape == (self.dataset.shape[1], self.dataset.shape[1])
-        npt.assert_array_almost_equal(oz.ranks_, expected)
+        assert oz.ranks_.shape == (X.shape[1], X.shape[1])
+        npt.assert_array_almost_equal(oz.ranks_, expected, decimal=5)
```

Comment:
Is the decimal parameter related to the numba issue we were having on miniconda?

Reply:
I'm not sure; I was noticing local failures with pytest with the default parameter value for `decimal` (which is 6), which were resolved by considering only 5 decimal places.

Reply:
Ah, ok - if it was local then it's not related. 5 decimal points are perfectly reasonable for array almost equal in this case; I just wanted to collate and collect related issues.